cropper/src/main/AndroidManifest.xml

```diff
@@ -1,4 +1,15 @@
-<manifest
-    package="com.canhub.cropper">
+<manifest package="com.canhub.cropper"
+    xmlns:android="http://schemas.android.com/apk/res/android">
 
+    <application>
+        <provider
+            android:name="androidx.core.content.FileProvider"
+            android:authorities="${applicationId}.provider"
+            android:exported="false"
+            android:grantUriPermissions="true">
+            <meta-data
+                android:name="android.support.FILE_PROVIDER_PATHS"
+                android:resource="@xml/provider_paths" />
+        </provider>
+    </application>
```

Comment:
Where are you using the authority?

Reply:
private void dispatchTakePictureIntent() {

        Intent takePictureIntent = new Intent(MediaStore.ACTION_IMAGE_CAPTURE);
        takePictureIntent.resolveActivity(this.getPackageManager());
        File photoFile = new File(getExternalFilesDir(Environment.DIRECTORY_PICTURES), "pickImageResult.jpeg");
        photoURI = FileProvider.getUriForFile(this, getApplicationContext().getPackageName() + ".fileprovider", photoFile);
        takePictureIntent.putExtra(MediaStore.EXTRA_OUTPUT, photoURI);
        startActivityForResult(takePictureIntent, REQUEST_TAKE_PHOTO);
    }

Reply:
Thanks @akkie2106 cannot find this in the code, could point me the class that is using it?

Reply:
photoURI = FileProvider.getUriForFile(this, getApplicationContext().getPackageName() + ".fileprovider", photoFile)
i'm using in this line because of the fileprovider im getting the photoURI

Reply:
but didn't found a solution for the camera image in android 11 still getting intent data null
@Canato

Reply:
If you are using this line in your project you need to add this provider into your project. 

I cannot see where ravindu is using authority on his PR, unless we add, is not something we need now, cause we need to change `getExternal` methods in the whole library, not only outside it. 

My comment is related to the library only, not each project which consume it. 😄

Reply:
@Canato Removed the Authority. We don't really need that.